spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -203,15 +203,49 @@ module Bulkrax
         it 'succeeds' do
           metadata = subject.build_metadata
           expect(metadata['multiple_objects'][0]['first_name']).to eq('Fake')
-          expect(metadata['multiple_objects'][0]['last_name']).to eq('Fakerson')
+          expect(metadata['multiple_objects'][0]['last_name']).to eq('')
           expect(metadata['multiple_objects'][0]['position']).to include('Leader', 'Jester', 'Queen')
           expect(metadata['multiple_objects'][0]['language']).to eq('English')
           expect(metadata['multiple_objects'][1]['first_name']).to eq('Judge')
           expect(metadata['multiple_objects'][1]['last_name']).to eq('Hines')
           expect(metadata['multiple_objects'][1]['position']).to include('King', 'Lord', 'Duke')
         end
-        # rubocop:enable RSpec/ExampleLength
       end
+
+      context 'with object fields prefixed and properties with multiple values' do
+        let(:importer) do
+          FactoryBot.create(:bulkrax_importer_csv, field_mapping: {
+                              'multiple_objects_first_name' => { from: ['multiple_objects_first_name'], object: 'multiple_objects' },
+                              'multiple_objects_last_name' => { from: ['multiple_objects_last_name'], object: 'multiple_objects' },
+                              'multiple_objects_position' => { from: ['multiple_objects_position'], object: 'multiple_objects', nested_type: 'Array' }
```

Comment:
setting `nested_type: 'Array'` instead of something like `array: true` so it allows for handling other nested types (e.g. object) in the future. if that ever becomes a need.